pkg/logging/sftp/sftp_test.go

```diff
@@ -0,0 +1,255 @@
+package sftp
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/cli/pkg/compute/manifest"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/mock"
+	"github.com/fastly/cli/pkg/testutil"
+	"github.com/fastly/go-fastly/fastly"
+)
+
+func TestCreateCreateInput(t *testing.T) {
```

Comment:
minor, but looks like there may be a bug in your code generation scripts:
```suggestion
func TestCreateSFTPInput(t *testing.T) {
```